index.html

```diff
@@ -4373,20 +4372,18 @@ <h3 id=shadow-root>Shadow Roots</h3>
  <p>An ECMAScript <a>Object</a> <dfn>represents a shadow root</dfn>
   if it has a <a>shadow root identifier</a> <a>own property</a>.
  
- <p>Each <a>browsing context</a> has an associated <dfn>list of
+ <p>Each <a>Window</a> has an associated <dfn>list of
```

Comment:
nit: is there a reason to have a capital letter here?

Reply:
It's called `Window` in HTML, see https://html.spec.whatwg.org/#the-window-object